rasa/core/actions/forms.py

```diff
@@ -428,6 +441,17 @@ def _temporary_tracker(
             slots=domain.slots,
         )
 
+    async def _user_rejected_manually(self, validation_events: List[Event]) -> bool:
+        """Returns True if slot validation rejected the form execution, else False.
```

Comment:
optional: at this point it might make more sense to use a fullblown docstring which includes `Args:` and `Returns`
```suggestion
        """Returns `True` if slot validation rejected the form execution, else `False`.
```

Reply:
Created a fullblow docstring which includes Args & Returns.